ansible/files/logrotate-postgres-csv

```diff
@@ -0,0 +1,11 @@
+/var/lib/postgresql/data/pg_log/postgresql.csv {
```

Comment:
We will need to roll this out to existing pg13 images too. I see logrotate is currently rotating `/var/lib/postgresql/data/pg_log/postgresql.log` which has no logs going to it

Reply:
Yeah there'll be a few more changes as well